src/main/java/janggi/domain/move/Vectors.java

```diff
@@ -1,34 +1,46 @@
 package janggi.domain.move;
 
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
 
 public record Vectors(List<Vector> vectors) {
 
-    public static Vectors of(Movement... movements) {
-        List<Vector> vectorList = Arrays.stream(movements)
+    public static Vectors of(Movement movement, Movement... remainMovements) {
```

Comment:
가변인자보다는 풍부한 기능을 제공하는 List 자료구조를 권장드립니다 😄 

```suggestion
    public static Vectors of(Movement movement, List<Movement> remainMovements) {
```

Reply:
해당 부분에 대해 고민을 하였는데 List를 사용하게 된다면 List.of를 너무 많이 사용하게 되는 거 같아서 우선은 보류하겠습니다.. 
변경을 하게 된다면 다음과 같은 구조가 되서 지금 구조에서는 가변인자를 사용하는게 가독성이 높다고 판단됩니다.

```java

    @Override
    protected List<Vectors> getVectorsList() {
        return List.of(
                Vectors.of(
                        List.of(Movement.DOWN, Movement.LEFT_DOWN, Movement.LEFT_DOWN)
                ),
                Vectors.of(
                        List.of(Movement.DOWN, Movement.RIGHT_DOWN, Movement.RIGHT_DOWN)
                )
        );
    }


```